docs/csharp/snippets/properties/Person.cs

```diff
@@ -7,7 +7,7 @@ namespace VersionOne
     // <Snippet1>
     public class Person
     {
-        public string FirstName;
+        public string? FirstName;
```

Comment:
Note for me to create an issue.

I think this article should be updated to make use of the `required` keyword in some of the examples. It's outside the scope of this PR, and would affect the text as well.